leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.kt

```diff
@@ -101,6 +102,7 @@ private const val DEFAULT_ZOOM = 16
 private const val WIDER_ZOOM = 6
 private const val MIN_CHARACTERS = 2
 private const val DEBOUNCE_TIME = 400
+private const val UNNAMED_ROAD = "Unnamed Road, "
```

Comment:
"Unnamed Road - " as well 😬 

It was tested on 6.2.0 version

![56770449_2538857246341424_643806844527050752_n](https://user-images.githubusercontent.com/31210460/56061955-4dbcff80-5d41-11e9-8975-e707bb3f333e.png)

Reply:
I fixed 🤠 

![image](https://user-images.githubusercontent.com/31210460/56084097-74903a00-5e04-11e9-98c9-d599b758c7e6.png)

Reply:
https://github.com/SchibstedSpain/Leku/pull/244